tests/test_keeper_chief.py

```diff
@@ -108,3 +114,50 @@ def test_check_hat(self, mcd: DssDeployment, keeper: ChiefKeeper, guy_address: A
         assert self.spell.eta() != 0
 
 
+    def test_check_eta_receipt(
+        self, mcd: DssDeployment,
+        keeper: ChiefKeeper,
+        simpledb: SimpleDatabase,
+        our_address: Address
+    ):
```

Comment:
For consistency with the other tests, please keep the test method interface to a single line.

Reply:
or, change them all to fit within 80 characters. 😏

Reply:
style revert